src/commands/site-list.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace Team51\Command;
+
+use Team51\Helper\API_Helper;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Helper\ProgressBar;
+
+class Site_List extends Command {
+	protected static $defaultName = 'site-list';
+
+	protected function configure() {
+		$this
+		->setDescription( 'Shows list of public facing sites managed by Team 51.' )
+		->setHelp( 'Use this command to show a list of sites and summary counts managed by Team 51.' );
+	}
+
+	protected function execute( InputInterface $input, OutputInterface $output ) {
+		$api_helper = new API_Helper;
+
+		$output->writeln( '<info>Fetching sites, approx. 2 minutes...<info>' );
```

Comment:
@nickpagz I learned the other day that specifying the fields you need on `/me/sites/` accelerates the query a lot.

This should do the trick to speed up from 2 minutes to 8 seconds:
```
$all_sites = $api_helper->call_wpcom_api( 'rest/v1.1/me/sites?fields=ID,name,URL,is_private,is_coming_soon,is_wpcom_atomic,jetpack', array() );
```

Reply:
@nickpagz everything works as expected on my end.

Please feel free to merge after the performance change is in place and the branch conflict is solved.

Reply:
Oh wow! Thanks! I'll take a look.